### PR TITLE
Add: CI / CD

### LINKED ISSUE #10: CI/CD
Add sz.rs into CI / CD

### PR DESCRIPTION
**Pull request by Jakunot**
Closes #10

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Install Rust
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        override: true
+
+    - name: Run tests
```
Comment (21st-centuryman) [inline .github/workflows/test.yml:20]: Change to Get code size

**Review by 21st-centuryman — CHANGES_REQUESTED** [review]
Added requests

File: .github/workflows/test.yml (hunk 1 above)
Comment (Jakunot) [reply .github/workflows/test.yml]: https://github.com/teeny-kth/teenygrad/pull/11/commits/ec5112ca3c2129cf0d52180bb13d2f7c2379847f

**Review by 21st-centuryman — COMMENTED** [review]
LGTM

**Review by 21st-centuryman — APPROVED** [review]
LGTM

**Pull request merged**